app/src/Application.php

```diff
@@ -99,11 +105,53 @@ public function middleware(MiddlewareQueue $middlewareQueue): MiddlewareQueue
             // https://book.cakephp.org/4/en/security/csrf.html#cross-site-request-forgery-csrf-middleware
             ->add(new CsrfProtectionMiddleware([
                 'httponly' => true,
-            ]));
+            ]))
+
+            ->add(new RoutingMiddleware($this))
```

Comment:
The RoutingMiddleware had been added earlier at this LOC https://github.com/sergio9977/tutorial-cakephp/blob/0485181bf1cb967d4ad52bcfdbd836f24ad27511/app/src/Application.php#L93-L97

Is it OK to add it twice?